style.css

```diff
@@ -301,3 +313,97 @@
     margin-top: 100px;
   }
 }
+
+@media screen and (min-width: 768px) {
+  .header {
+    justify-content: center;
+  }
+
+  .header .links {
+    display: block;
+    margin: 0 auto;
+  }
+
+  .header .links a {
+    color: #172b4d;
+    padding: 10px 12px;
+    text-decoration: none;
+  }
+
+  .head-line > button {
+    border: 1px solid #dfe1e6;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    padding: 8px;
+  }
+
+  .header > h2,
+  .header > nav button {
+    display: none;
+  }
+
+  .head-line {
+    padding: 160px 0;
+    background-image: url(./media/Header-llustration-desktop@2x.png);
+  }
+
+  .description {
+    width: 920px;
+  }
+
+  .work-container {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 474px);
+    column-gap: 15px;
+  }
+
+  .about-container {
```

Comment:
Optional.
Consider using the same font color as in the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/192796228-8b96d2a8-484e-4e7f-861c-98d51f2d2a80.png) | ![image](https://user-images.githubusercontent.com/33268927/192796429-1c4935c1-3681-48ea-a4c5-2547b583b80b.png) |